utilities_ubuntu_18.04/mv.cwl

```diff
@@ -0,0 +1,70 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: mv
+baseCommand:
+  - mv
+inputs:
+  - id: memory_per_job
+    type: int?
+    doc: Memory per job in megabytes
+  - id: memory_overhead
+    type: int?
+    doc: Memory overhead per job in megabytes
+  - id: infile
+    type: File
+    inputBinding:
+      position: 1
+    doc: 'Source with path that needs to be moved'
+  - id: outfile
+    type: string
+    inputBinding:
+      position: 2
+    doc: 'Target destination for the source'
+  - id: force
+    type: boolean?
+    inputBinding:
+      position: 0
+      prefix: '-f'
+    doc: 'Do not prompt for confirmation before overwriting the destination path.'
+  - id: verbose
+    type: boolean?
+    inputBinding:
+      position: 0
+      prefix: '-v'
+    doc: 'Cause mv to be verbose, showing files after they are moved.'
+outputs:
+  - id: out
+    type: File
+    outputBinding:
+      glob: $(inputs.outfile)
+label: mv
+requirements:
+  - class: ResourceRequirement
+    ramMin: 100
```

Comment:
unfortunately I think this will error with current version of Toil, which divides and then uses a floor on ramMin

so (1000 / 1024) --> 0 and lsf won't accept the job

the lowest value that I've given jobs is 2000

obviously this is not ideal

Reply:
a further fix to toil might be to modify lsf.py to get more fine-grained resource requests